packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/sign-the-csr/index.md

```diff
@@ -2,7 +2,22 @@
 title: Sign the CSR
 ---
 
-Follow the third-party process that your company uses to sign the CSR. You can't move to step four until you complete the process because the signed CSR is required to continue.
+Follow the third-party Certificate Authority (CA) process that your company uses to sign the CSR. You cannot proceed to the next step until your certificate is signed using this process.
+
+The following is a list of popular third-party CA providers that can be used to sign the CSR:
+ - Comodo SSL
+ - DigiCert
+ - GoDaddy
```

Comment:
GoDaddy should be after GlobalSign in alphabetical order

Reply:
made the change